src/box/applier.cc

```diff
@@ -323,21 +323,14 @@ apply_nop(struct xrow_header *row)
 	return process_nop(&request);
 }
 
-/**
- * Connect to a remote host and authenticate the client.
- */
-void
-applier_connect(struct applier *applier)
+/** Common part of connection initialization. */
+static void
+applier_connection_init(struct iostream *io, const struct uri *uri,
```

Comment:
Nit (up to you): hm, this looks almost exactly like `netbox_transport_connect()`. Maybe make a common function for both places?

Reply:
Let's please leave it as is for now. I'm not into any more refactoring right now :)